src/main/java/com/tdd/hhplus_tdd_2week_java/domain/lecture/LectureRepository.java

```diff
@@ -4,5 +4,4 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface LectureRepository extends JpaRepository<Lecture, Long>, LectureRepositoryCustom {
```

Comment:
Domain 레이어에서 JPA를 의존 하는것 보단 DIP를 통해 비즈니스 로직에 필요한 메서드만 갖는 순수한 인터페이스를 만들고 Infrastructure 레이어에 구현을 하는것이 좋아보입니다.

추가로 설명 필요하시다면 DM 주시면 설명해 드리겠습니다!
```suggestion
/domain
    LectureRepository.java
/infra
    LectureRepositoryImpl.java
        jpa, queryDsl 의존하여 구현
```